rosco-web/src/main/groovy/com/netflix/spinnaker/rosco/controllers/BakeryController.groovy

```diff
@@ -182,11 +181,15 @@ class BakeryController {
         }
       }
 
-      def packerCommand = cloudProviderBakeHandler.producePackerCommand(region, bakeRequest)
-      def jobRequest = new JobRequest(tokenizedCommand: packerCommand, maskedPackerParameters: cloudProviderBakeHandler.maskedPackerParameters, jobId: bakeRequest.request_id)
+      def bakeRecipe = cloudProviderBakeHandler.produceBakeRecipe(region, bakeRequest)
+      def jobRequest = new JobRequest (
```

Comment:
I do see that we've been inconsistent with our formatting already. I prefer this style, but can live with it either way.